org_fedora_oscap/common.py

```diff
@@ -309,6 +310,46 @@ def run_oscap_remediate(profile, fpath, ds_id="", xccdf_id="", tailoring="",
     return proc.stdout
 
 
+def _schedule_firstboot_remediation(
+        chroot, profile, ds_path, results_path, report_path, ds_id, xccdf_id, tailoring_path):
+    config = textwrap.dedent(f"""\
+    OSCAP_REMEDIATE_DS='{ds_path}'
+    OSCAP_REMEDIATE_PROFILE_ID='{profile}'
+    OSCAP_REMEDIATE_ARF_RESULT='{results_path}'
+    OSCAP_REMEDIATE_HTML_REPORT='{report_path}'
+    """)
+    if ds_id:
+        config += "OSCAP_REMEDIATE_DATASTREAM_ID='{ds_id}'\n"
```

Comment:
ouch, we miss the tiny, but important letter `f` here and on lines below.

Reply:
Then how its possible that it worked? What did I test?